documentation/pysa_tutorial/exercise2/.pyre_configuration

```diff
@@ -10,5 +10,6 @@
   ],
   "exclude": [
     ".*/integration_test/.*"
-  ]
+  ],
+  "use_command_v2": true
```

Comment:
I'm not opposed to committing this, but if we do it should be in a separate PR and should affect all exercises